src/scala/scripts/BUILD

```diff
@@ -21,3 +21,29 @@ scala_binary(
         ":generator_lib",
     ],
 )
+
+scala_library(
+    name = "scalapb_generator_lib",
+    srcs = ["ScalaPBGenerator.scala"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//external:io_bazel_rules_scala/dependency/proto/protoc",
+        "//external:io_bazel_rules_scala/dependency/proto/scalapb_plugin",
+        "//external:io_bazel_rules_scala/dependency/proto/protoc_bridge",
+        "//external:io_bazel_rules_scala/dependency/proto/scalapbc",
+        "//external:io_bazel_rules_scala/dependency/com_google_protobuf/protobuf_java",
+        "//src/java/io/bazel/rulesscala/io_utils",
+        "//src/java/io/bazel/rulesscala/jar",
+        "//src/java/io/bazel/rulesscala/worker",
+        "//src/scala/io/bazel/rules_scala/scrooge_support:compiler",
```

Comment:
Why does protobuf generator need to depend on scrooge?

Reply:
Fix is here: https://github.com/bazelbuild/rules_scala/pull/290/files